db/migrate/20200119195227_add_visited_path_to_pageviews.rb

```diff
@@ -0,0 +1,7 @@
+class AddVisitedPathToPageviews < ActiveRecord::Migration[5.2]
+  def change
+    add_column :page_views, :visited_page_path,     :string
```

Comment:
Given the name of the table is `page_views` you could probably remove `visited_page_` from these column names if you wanted.

Reply:
The problem is we already have a `path` column and we use it for the _referrer_ path. 🤦‍♂️

We could use this as a chance to rename that column, but that strikes me as a heavy-duty migration given the size of the table.

Reply:
You can just rename the column with `ALTER TABLE table_name  RENAME COLUMN old_name TO new_name;` but it's still something to be careful about, as that statement will take an `AccessExclusive` lock on the table it modifies. Even if this can seems fast in a local/staging env, it can lead to problems in production, see e.g. [this post](https://gocardless.com/blog/zero-downtime-postgres-migrations-the-hard-parts/).

Reply:
As @citizen428 said, renaming columns is not the best operation for tables, the documentation for the new gem `strong-migrations` has a good plan of attack related to renaming columns: https://github.com/ankane/strong_migrations#renaming-or-changing-the-type-of-a-column

Reply:
I did not realize we already had a column, yeah renaming would be a bitch and not worth it in this case IMO